.github/workflows/deploy.yml

```diff
@@ -42,8 +42,6 @@ jobs:
       - name: Deploy with kamal
         run: |
           kamal lock release
-          kamal accessory boot yosys2digitaljs-server
-          kamal traefik boot
```

Comment:
Unrelated

Reply:
we can remove this as we have already booted those containers.

Reply:
Here you can see, this is my personal server:
If we run this again then that container already exists. Hence the action will fail.

![test-servre](https://github.com/CircuitVerse/CircuitVerse/assets/85568177/cd7333c1-429a-4274-b1b6-0d986ce3e04a)

Reply:
Sorry, merged the PR without checking this thread. Our deployment script should be smart enough to check if it is already booted or not.